apis/bases/core.openstack.org_openstackcontrolplanes.yaml

```diff
@@ -16499,20 +16499,84 @@ spec:
                 type: object
               tls:
                 default:
-                  endpoint:
-                    internal:
-                      enabled: false
-                    public:
-                      enabled: true
+                  ingress:
+                    ca:
+                      duration: 43800h
+                    cert:
+                      duration: 8760h
+                    enabled: true
+                  podLevel:
+                    default:
+                      ca:
+                        duration: 43800h
+                      cert:
+                        duration: 8760h
+                    enabled: false
+                    ovn:
+                      ca:
+                        duration: 43800h
+                      cert:
+                        duration: 8760h
                 properties:
                   caBundleSecretName:
                     type: string
-                  endpoint:
-                    additionalProperties:
-                      properties:
-                        enabled:
-                          type: boolean
-                      type: object
+                  ingress:
+                    properties:
+                      ca:
+                        properties:
+                          duration:
+                            type: string
+                          renewBefore:
+                            type: string
+                        type: object
+                      cert:
+                        properties:
+                          duration:
+                            type: string
+                          renewBefore:
+                            type: string
+                        type: object
+                      enabled:
+                        type: boolean
+                    type: object
+                  podLevel:
+                    properties:
+                      default:
```

Comment:
kindof implies that ovn will default to this too but IIUC it will not

Reply:
as discussed, renamed to internal